requirements/base.txt

```diff
@@ -27,4 +27,5 @@ South==1.0.1
 voluptuous==0.8.5
 
 # AI grading
-git+https://github.com/edx/ease.git@b67d2928a26fe497826b6ea359b9a3d0371548a7#egg=ease==0.1.3
+git+https://github.com/edx/ease.git@release-2015-07-09#egg=ease==0.1.3
+-e git+https://github.com/edx/nltk.git@2.0.6#egg=nltk
```

Comment:
Instead of `-e` let's the same format as ease above of actually pinning to the version.